package.json

```diff
@@ -79,27 +79,26 @@
     "is-promise": "^2.1.0",
     "isomorphic-fetch": "^2.2.1",
     "js-yaml": "^3.13.1",
-    "material-ui": "^0.16.5",
+    "material-ui": "0.20.1",
     "moment": "^2.20.1",
     "moment-timezone": "^0.5.14",
     "normalizr": "^2.2.0",
     "path-complete-extname": "^0.1.0",
     "postcss-loader": "^2.0.10",
     "postcss-smart-import": "^0.7.6",
-    "prop-types": "^15.6.0",
+    "prop-types": "^15.7.2",
     "ramda": "^0.23.0",
     "react": "^15.6.2",
     "react-apollo": "^2.0.4",
-    "react-big-calendar": "^0.12.3",
+    "react-big-calendar": "0.17.0",
     "react-dom": "^15.6.2",
     "react-geosuggest": "^2.7.0",
     "react-google-maps": "^4.11.0",
     "react-i18next": "^8.3.9",
     "react-redux": "^4.4.5",
-    "react-router": "^2.6.0",
+    "react-router": "^3.2.5",
```

Comment:
Would there be a fix for breaking changes migrating to v3?

The save button does not work when adding new events in the admin view when using v3 (http://localhost:5000/portal/admin/events/new)

The events still get created, but the page does not get redirected to the dashboard.

Reply:
is it that `props.history.push` is deprecated in favour of using the new `this.context.history`

Reply:
`history.push` is still valid in React Router v5.

Reply:
Thanks for that @Pancrisp. I had heard there was something that broke in v3 in the app but wasn't sure which. And yeah the changelog when I looked didn't identify anything suspicious. 

I'll hold off on merging this PR and do some debugging when I'm working on this next Wednesday.

Reply:
I've modified `props.history.push` to `props.router.push` and it appears to be working now.

Reply:
great news :)